xplenty/xplenty_api.py

```diff
@@ -327,14 +348,28 @@ def create_cluster(self, cluster_type, nodes, cluster_name, cluster_description,
         cluster_info["cluster[description]"]= cluster_description if cluster_description else ""
         cluster_info["cluster[terminate_on_idle]"]= 1 if terminate_on_idle else 0
         cluster_info["cluster[time_to_idle]"]= time_to_idle
+        #New entries available
+        cluster_info["cluster[region]"]=region if region else ""
```

Comment:
@xomri - Empty region might fail the call. So either remove the `else ""` or give it a default value.